hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/queries/validation/QueryFeeCheck.java

```diff
@@ -162,21 +161,6 @@ public ResponseCodeEnum validateQueryPaymentTransfers(final TransactionBody txn)
         return OK;
     }
 
-    public com.hedera.hapi.node.base.ResponseCodeEnum validateQueryPaymentTransfers2(
```

Comment:
Does it makes sense to deprecate this class?

Reply:
No, the class is still being used. The mono-service uses it and hedera-app delegates its call. All I did was to extract these methods into an adapter.

Reply:
the class is still being used <- I do not ask do delete it ;) But by adding the @Deprecated annotation and a short message other developers will know about the new facility and do not use the old classes in new / refactored code

Reply:
Ah, right. Sorry, misunderstood you.

It is not so much about old code vs. new code. Code in `mono-service` should continue to use this class, code in `hedera-app` should use the new class. Adding the `@Deprecated` annotation would be confusing, if you are working within mono-service.